www/docs/usage/config-files.md

```diff
@@ -36,49 +42,66 @@ terraform {
+If you do not have write access to `/usr/local/bin/` directory, you can use the
+`.tfswitch.toml` file to specify a **full installation path** (directory +
+filename) Ex: `$HOME/bin/terraform`  
+This is similar to using a `.tfswitchrc` file, but you specify a custom binary
+path for the installation:
 
 ![toml1](../static/tfswitch-v7.gif)
 ![toml2](../static/tfswitch-v8.gif)
 
 1. Create a directory for the custom binary path. Ex: `mkdir -p "$HOME/bin/"`
-2. Add the path to the directory to your `PATH` environment variable. Ex: `export PATH="$PATH:$HOME/bin"` (add this to your Bash profile or Zsh profile)
-3. Pass `-b` or `--bin` parameter with the custom binary path as value (this must be a first level pointer inside the directory from above). Ex: `tfswitch -b "$HOME/bin/terraform" 0.10.8`
-   - If target directory for custom binary path does not exist, `tfswitch` falls back to `$HOME/bin/` directory
-4. Optionally, you can create a `.tfswitch.toml` file in your home directory (`~/.tfswitch.toml`)
-5. Your `~/.tfswitch.toml` file should look like this:
-
-```toml
-bin = "$HOME/bin/terraform"
-version = "0.11.3"
-```
-
-6. Run `tfswitch` and it should automatically install the required version in the specified binary path
+2. Add the path to the directory to your `PATH` environment variable. Ex:
+   `export PATH="$PATH:$HOME/bin"` (add this to your Bash profile or Zsh
+   profile)
+3. Pass `-b` or `--bin` parameter with the custom binary path as value (this
+   must be a first level pointer inside the directory from above). Ex:
+   `tfswitch -b "$HOME/bin/terraform" 0.10.8`  
+   • If target directory for custom binary path does not exist, `tfswitch`
+   falls back to `$HOME/bin/` directory
+4. Optionally, you can create a `.tfswitch.toml` file in your home directory
+   (`~/.tfswitch.toml`)
+5. Your `~/.tfswitch.toml` file should have a line like this:
+   `bin = "$HOME/bin/terraform"`
+6. Run `tfswitch` and it should automatically install the required version in
+   the specified binary path
 
 Below is an example for `$HOME/.tfswitch.toml` on Windows:
 
 ```toml
 bin = "C:\\Users\\<%USRNAME%>\\bin\\terraform.exe"
 ```
 
-## Setting the default (fallback) version using `.tfswitch.toml` file
+### Setting default (fallback) version
 
-By default, if `tfswsitch` is unable to determine the version to use, it errors out.  
-The `.tfswitch.toml` file can be configured with a `default-version` attribute for `tfswitch` to use a particular version, if no other sources of versions are found
+By default, if `tfswsitch` is unable to determine the version to use, it errors
+out.  
+The `.tfswitch.toml` file can be configured with a `default-version` parameter
+for `tfswitch` to use a particular version, if no other sources of versions are
+found
 
 ```toml
 default-version = "1.5.4"
 ```
 
-## Setting product using `.tfswitch.toml` file
+### Setting product (`terarform` or `opentofu`)
```

Comment:
I'm not sure about having the valid options in the title, as if we support a third product, it would break all of the links.
Also, typo in `terraform` :)